actions/image_partition_action.go

```diff
@@ -613,6 +614,27 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 			}
 		}
 
+		if i.PartitionType == "gpt" {
+			var maxLength int = 0
+			if p.PartLabel == "" {
+				p.PartLabel = p.Name
+			}
+
+			switch p.FS {
```

Comment:
also, can we have a look at the maximum lengths for the other filesystems ?

Reply:
Sure. Will check for btrfs, xfs, reiserfs, jfs.